samples/csharp/MicrosoftEduImportFromGoogle/Import.cs

```diff
@@ -0,0 +1,323 @@
+﻿using Microsoft.Extensions.Configuration;
+using Microsoft.Graph.Beta;
+using Microsoft.Graph.Beta.Models;
+using MicrosoftEduImportFromGoogle.Models;
+using MicrosoftGraphSDK;
+
+namespace MicrosoftEduImportFromGoogle
+{
+    /// <summary>
+    /// Microsoft endpoints needed to import assignments, classwork modules and resources
+    /// </summary>
+    internal class Import
+    {
+        private readonly IConfiguration _config;
+        public GraphServiceClient graphServiceClient;
+        public Import(IConfiguration configuration)
+        {
+            this._config = configuration;
+        }
+
+        /// <summary>
+        /// Authorizes the application and creates a Microsoft Graph client
+        /// </summary>
+        /// <returns></returns>
+        public void AuthorizeApp()
+        {
+            // App-only scenario
+            this.graphServiceClient = GraphClient.GetApplicationClient(_config["microsoftTenantId"], _config["microsoftClientId"], _config["microsoftSecret"]);
+
+            // Use the line below for App+user scenario
+            // this.graphServiceClient = GraphClient.GetDelegateClient(_config["microsoftClientId"]);
+        }
+
+        /// <summary>
+        /// Returns a list of Teams that the requesting user is permitted to view
+        /// </summary>
+        /// <param name="userId">User id</param>
+        /// <returns>List<Microsoft.Graph.Beta.Models.Team></returns>
+        public List<Microsoft.Graph.Beta.Models.Team> GetUserClasses(string userId)
+        {
+            Console.WriteLine("* Fetching classes from Microsoft Teams...");
+            return graphServiceClient
+                    .Users[userId]
+                    .JoinedTeams
+                    .GetAsync()
+                    .Result.Value.ToList();
+        }
+
+        /// <summary>
+        /// Returns a list of classes that the requesting application is permitted to view
+        /// </summary>
+        /// <returns>List<EducationClass></returns>
+        public List<EducationClass> GetClasses()
+        {
+            Console.WriteLine("* Fetching classes from Microsoft Teams...");
+            return graphServiceClient.Education.Classes
+                    .GetAsync()
+                    .Result.Value.ToList();
+        }
+
+        /// <summary>
+        /// Maps Google entities to create assignments
```

Comment:
Would it better to call this out as Map {GoogleEntityName} to {MSEntityName}? Same for all the Map methods below